cura/MachineActionManager.py

```diff
@@ -105,7 +105,7 @@ def getSupportedActions(self, definition_id):
         if definition_id in self._supported_actions:
             return list(self._supported_actions[definition_id])
         else:
-            return set()
+            return list()
```

Comment:
You can also add a return type to the function definition. That way, mypy can also find this error.